elements/lisk-p2p/src/peer.ts

```diff
@@ -475,6 +482,9 @@ export class Peer extends EventEmitter {
 			multiplex: false,
 			autoConnect: false,
 			autoReconnect: false,
+			wsEngineServerOptions: {
+				maxPayload: this._peerConfig.wsMaxPayload,
+			},
```

Comment:
This will not be used. For the client, we should use `maxPayload: this._peerConfig.wsMaxPayload` as a top level property otherwise it will be ignored. The approach above is only for the server-side.